.editorconfig

```diff
@@ -11,7 +11,7 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Comment:
Can you reset this? `2` is what the default blueprint generates, and is much more common.

Reply:
Yep, will do this afternoon. Thanks!